astro-client-core/api.gen.go

```diff
@@ -234,11 +234,23 @@ const (
 	CreateEnvironmentObjectLinkRequestScopeDEPLOYMENT CreateEnvironmentObjectLinkRequestScope = "DEPLOYMENT"
 )
 
+// Defines values for CreateEnvironmentObjectMetricsExportOverridesRequestAuthType.
```

Comment:
The updates in this file seem all unrelated to remote execution and its enablement?

Reply:
I guess came along for the ride for free when we updated platform-core?

Reply:
Correct. I have generated it from the release branch that targeted the v1beta1 platform support.